src/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GeoJson\Exception;
+
+use InvalidArgumentException as InternalInvalidArgumentExceptionAlias;
+
+class InvalidArgumentException extends InternalInvalidArgumentExceptionAlias implements Exception
```

Comment:
This file can remain as it was:

```php
<?php

declare(strict_types=1);

namespace GeoJson\Exception;

class InvalidArgumentException extends \InvalidArgumentException implements Exception
{
}
``

Please make the following change in `phpcs.xml.dist`:

```diff
-<property phpcs-only="true" name="allowFullyQualifiedNameForCollidingClasses" value="false"/>
<property phpcs-only="true" name="allowFullyQualifiedNameForCollidingClasses" value="true"/>
```